pages/plugins.vue

```diff
@@ -1,20 +1,81 @@
 <template>
   <div class="container">
     <PluginsNav />
-    <div class="tutorial-markdown-window">
-      <Plugins />
+    <div class="family-grid-wrapper">
+      <h1 class="hapi-header">Plugins</h1>
+      <p>
+        There are dozens of plugins for hapi, ranging from documentation to authentication,
+        and much more. If you wrote or use a plugin that you don't see on this list, please
+        send us a <a href="https://github.com/hapijs/hapi.dev/blob/master/static/lib/plugins.md">pull request</a>.
+      </p>
+      <p>
+        <img src="/img/family.svg" class="plugins-logo-top">
+        -   Designates an offical hapi family plugin
+      </p>
+      <div :id="category.name + 1"
```

Comment:
There is a mismatch here between `category.name` value and the anchor used in [PluginNav.vue](https://github.com/hapijs/hapi.dev/blob/master/components/plugins/PluginsNav.vue) for the categories with a `/` in it. For example for the category `Logging/Metrics` the anchor in the menu has `href="#logging"` whereas `:id="category.name + 1"` results in `id="Logging/Metrics1"` hence the menu link does not work.

Reply:
Fixed so that the categories define the anchor, also updated so that the nav is generated from the static plugins json